test_cases/test_rooms.py

```diff
@@ -1,30 +1,33 @@
 import unittest
-from app.room import OfficeSpace, LivingSpace
+from ..app.room import OfficeSpace, LivingSpace
 
 
 class TestCreateRoom(unittest.TestCase):
 	"""Test room creation """
 	def setUp(self):
-		self.office_space = OfficeSpace()
-		self.living_space = LivingSpace()
+		self.office_space = OfficeSpace("Red")
+		self.living_space = LivingSpace("Emerald")
 	
 	def test_officespace_size(self):
-		self.assertEqual(self.office_space.size , 6 ,
+		self.assertEqual(self.office_space.capacity, 6,
 			msg ='Maximum capacity is 6 occupants')
 
-	def test_officespace_type(self)
+	def test_officespace_type(self):
 		self.assertEqual(self.office_space.room_type, "OFFICE",
 			msg = "Room created should be of type OFFICE")
 
-	def test_officespace_rejects_int(self):
-		self.assertRaises(ValueError, self.office_space.name, 111)
-
 	def test_livingspace_size(self):
-		self.assertEqual(self.living_space.size, 4, msg = 'Maximum capacity is 4 occupants')
+		self.assertEqual(self.living_space.capacity, 4, 
+			msg = 'Maximum capacity is 4 occupants')
 
-	def test_livingspace_type(self)
+	def test_livingspace_type(self):
 		self.assertEqual(self.living_space.room_type, "LIVING SPACE",
 			msg = "Room Created should be of type LIVING SPACE")
 	
-	def test_livingspace_rejects_int(self):
-		self.assertRaises(ValueError, self.living_space.name, 111)
+	def test_allowed_maximum_number_of_occupants_per_office(self):
```

Comment:
Your test cases are very descriptive. This sounds better to me. `test_the_maximum_number_of_occupants_allowed_per_office`
Please let me know what you think.